### DISCUSSION TITLE
Adding a background image and adapting it to any screen

### DISCUSSION THREAD

**Discussion by Andrukva** [twbs/bootstrap#34076] [Q&A]
Good people, please explain how to correctly add a picture to the background of the site and make it adapt to any screen. Already eaten all the baldness. Show on some example please. I would be very grateful for your help!

**Comment by GeoSot** [comment]
I think you search for this:

```css
background-size: cover;
background-repeat: no-repeat;
```

**Comment by ascensus-mdb** [comment]
Here you can find nice solutions and multiple examples https://mdbootstrap.com/docs/standard/content-styles/background-image/
